src/docfx/Models/BuildJsonConfig.cs

```diff
@@ -123,5 +123,8 @@ public class BuildJsonConfig
 
         [JsonProperty("customLinkResolver")]
         public string CustomLinkResolver { get; set; }
+
+        [JsonProperty("versions")]
+        public Dictionary<string, VersionInfo> Versions { get; set; }
```

Comment:
FileMappingItem uses its VersionName as key here to fetch the VersionInfo

---
In reply to: [102691611](https://github.com/dotnet/docfx/pull/1342#discussion_r102691611) [](ancestors = 102691611)